README.md

```diff
@@ -1,16 +1,71 @@
-<p align="center">
-   <img src="https://github.com/xetdata/pyxet/blob/0c7608c97f6a2a0cb2c83dd38fb717913c4d7522/docs/images/logo.png" alt="logo" width="400" />
-</p>
+# Git-Xet: Scale Git to Terabyte-sized Repos
 
-## What is it?
-xet-core is a collection of Rust crates which provide the low level, high performance, functionality and API bindings for [pyxet](https://pyxet.readthedocs.io/en/latest/) and [git-xet](https://xethub.com/assets/docs/getting-started/install). 
+Git Xet is a plugin for git that allows git repositories to scale to terabytes of data.  
 
-For more details, see [Git is for Data](https://www.cidrdb.org/cidr2023/papers/p43-low.pdf).
+## Design Goals
+
+### Binary Content Deduplication
```

Comment:
Is it important to call it 'Binary Content Deduplication'? Elsewhere we just talk about deduplication. I believe we can call out binary content in the description, but I wonder if the name as is may overcomplicate the narrative. You also then talk about working with repositories with large datasets that evolve over time in the description -- and one could argue that not all datasets use binary files. Thus it raises confusion as to whether this will help my scenario as I read through it.

Reply:
Cleared this up. Good thought!